.github/workflows/nightly-build.yml

```diff
@@ -47,30 +47,29 @@ jobs:
         working-directory: ./rust
     steps:
       - uses: actions/checkout@v4
-      - name: Setup Rust toolchain
-        uses: actions-rust-lang/setup-rust-toolchain@v1.9.0
-        with:
-          cache-workspaces: rust -> target
-          rustflags: ""
-      - name: Configure sccache
-        run: |
-          echo "RUSTC_WRAPPER=sccache" >> $GITHUB_ENV
-          echo "SCCACHE_GHA_ENABLED=true" >> $GITHUB_ENV
-      - name: Run sccache-cache
-        uses: mozilla-actions/sccache-action@v0.0.9
-      - name: Install dependencies
-        run: sudo apt-get update && sudo apt-get install -y protobuf-compiler
+      # TODO: Use actions/cache@v4 to cache target/ directory
```

Comment:
open an issue to track?

Reply:
https://github.com/numaproj/numaflow/issues/2644